src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -5,10 +5,13 @@
 # pylint: disable=line-too-long
 
 from argcomplete.completers import FilesCompleter
-from azure.cli.core.commands.parameters import get_location_type
+from azure.cli.core.commands.parameters import get_location_type, get_enum_type
 from azure.cli.core.commands.parameters import (file_type)
 from azure.cli.core.commands.validators import get_default_location_from_resource_group
 
+distribution_types = ["auto", "generic", "openshift", "rancher_rke", "kind", "k3s", "minikube", "gke", "eks", "aks", "aks_hci", "capz", "aks_engine", "tkg"]
```

Comment:
Can you import the enum values from consts.py?
See Example: https://github.com/Azure/azure-cli-extensions/blob/master/src/aks-preview/azext_aks_preview/_params.py

Reply:
Okay, will update accordingly.

Reply:
I will add the two lists in constants.py and import those in _params.py.